.reek.yml

```diff
@@ -1,31 +0,0 @@
----
```

Comment:
We had already disabled reek in #165, but this file stuck around. While tracking down the method complexity error CodeClimate error I found this and figured I'd remove it. Can do another PR if anybody feels strongly